applications/HDF5Application/python_scripts/create_xdmf_file.py

```diff
@@ -32,17 +32,20 @@ def GetSpatialGrid(h5py_file):
 
 
 def GetNodalResults(h5py_file):
-    nodal_results_path = "/ResultsData/NodalSolutionStepData"
+    nodal_results_paths = ["/ResultsData/NodalSolutionStepData", "/ResultsData/NodalDataValues"]
     results = []
-    results_group = h5py_file.get(nodal_results_path)
-    for variable_name in results_group.keys():
-        if isinstance(results_group[variable_name], h5py.Dataset):
-            data = xdmf.HDF5UniformDataItem(results_group.get(variable_name))
-            results.append(xdmf.NodalSolutionStepData(variable_name, data))
+    for nodal_results_path in nodal_results_paths:
+        if not nodal_results_path in h5py_file:
+            continue
+        results_group = h5py_file.get(nodal_results_path)
+        for variable_name in results_group.keys():
+            if isinstance(results_group[variable_name], h5py.Dataset):
+                data = xdmf.HDF5UniformDataItem(results_group.get(variable_name))
+                results.append(xdmf.NodalSolutionStepData(variable_name, data))
```

Comment:
I was thinking that, If someone wants to post-process data and visualize them, it would be better to have it in the xdmf. How would you prefer to rename it?

There will be a conflict of variable names if one uses same variable in the NodalSolutionStepData container and DataValue container, and wants to output both of them to xdmf.